etherman/feijoa_contracts.go

```diff
@@ -0,0 +1,38 @@
+package etherman
+
+import (
+	"github.com/0xPolygonHermez/zkevm-node/etherman/smartcontracts/feijoapolygonzkevm"
+	"github.com/0xPolygonHermez/zkevm-node/log"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// FeijoaContracts represents the contracts of the Feijoa upgrade
+type FeijoaContracts struct {
+	FeijoaZKEVMAddress common.Address
+	FeijoaZKEVM        *feijoapolygonzkevm.Feijoapolygonzkevm
+	//FeijoaRollupManager         *feijoapolygonrollupmanager.Feijoapolygonrollupmanager
+	//FeijoaGlobalExitRootManager *feijoapolygonzkevmglobalexitroot.Feijoapolygonzkevmglobalexitroot
+}
+
+// NewFeijoaContracts creates a new FeijoaContracts
+func NewFeijoaContracts(ethClient bind.ContractBackend, l1Config L1Config) (*FeijoaContracts, error) {
+	FeijoaZKEVMAddress := l1Config.RollupManagerAddr
+	FeijoaZKEVM, err := feijoapolygonzkevm.NewFeijoapolygonzkevm(FeijoaZKEVMAddress, ethClient)
```

Comment:
I'm not sure about this. Seems it is the wrong address

Reply:
You are right, I'll remove the old one

Reply:
Good catch!, I'll change it